### ISSUE TITLE
razorpage .net6

### ISSUE THREAD

**Issue by davut5** [Basaingeal/Razor.SweetAlert2#2009]
1- i installed package. 
2-add builder.services.  and in index page add using CurrieTechnologies.Razor.SweetAlert2; 
3- how can I use basically show message. 

public void OnPostSave()
        {
           Swal.FireAsync("Hello world!");
        }
Swal does not exist in the current context. :(

**Comment by Basaingeal** [comment]
Do you have this towards the top of your `.razor` file?

`@inject SweetAlertService Swal;`

**Comment by davut5** [comment]
I added top of the index.cshtml. but give me reference error.. :(

**Comment by davut5** [comment]
@Basaingeal

**Comment by Basaingeal** [comment]
It needs to go at the top of whichever `.razor` file is using it. So wherever your `OnPostSave()` call is written, it needs to be in that file.

**Comment by davut5** [comment]
this is my index.cshtml file  top line.. no error.
@inject SweetAlertService Swal;
@using CurrieTechnologies.Razor.SweetAlert2;

and this is my index.cs file  
public async Task<IActionResult> OnPostDogrula()
        {
await Swal.FireAsync("Hello world!");
}
gives me error. Swal does not current context.. :(

**Comment by Basaingeal** [comment]
I'll try to do a minimal replication and see what's going on there.

**Comment by davut5** [comment]
oh ok thank you. I think you put some sample. for Blazor, for Razorpage etc.

**Comment by w00dl3s** [comment]
I have the same issue. Did you already find a solution?

**Comment by Basaingeal** [comment]
I think I found the issue. I did not read the the title of the issue carefully enough.

I believe this library does not suite your use case. This is a library is only intended to work with Blazor applications (WASM or Server) , not with Razor Pages in a traditional ASP.NET web application.

Blazor is a front-end framework, and the code in this library runs on the front-end. Razor Pages are, instead, a way of templating server-generated pages.

I'll admit I have limited familiarity with Razor Pages, although I do believe there is a way to render Blazor components inside of them. Still this library doesn't offer any Blazor components. Just a Blazor services.

I will look into the possibility of SweetAlert being added to a RazorPage using this library, but for now, you'll want to use the main JS version of the library in the client section of your application.
